vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1279,6 +1264,11 @@ def fused_experts_impl(hidden_states: torch.Tensor,
                                         use_int4_w4a16=use_int4_w4a16,
                                         dtype=hidden_states.dtype)
 
+    qtype = get_config_qtype(use_fp8_w8a8=use_fp8_w8a8,
+                             use_int8_w8a8=use_int8_w8a8,
+                             use_int8_w8a16=use_int8_w8a16,
+                             use_int4_w4a16=use_int4_w4a16)
```

Comment:
Not for this PR, but we should seriously consider making an MoeConfig that can bundle together these and other various args that we pass through everywhere